openage/opus/setup.py

```diff
@@ -0,0 +1,14 @@
+# this file should be removed in the future.
+# asdf
+from distutils.core import setup
+from distutils.extension import Extension
+from Cython.Build import cythonize
+
+setup(
+    name = 'opusencoder',
+    ext_modules = cythonize([
+        Extension("opusenc",["opusenc.pyx"],
+                   libraries=["ogg","opus"])
+        ])
+)
+
```

Comment:
So I need to make a cmake module for finding libogg? The FindOpusfile.cmake seems to also find libopus.
side question: will I need to call `find_package(Opusfile)` again, since this is under `openage` and not under `libopenage/CMakeLists.txt`, where it is called first?